Homeworks/Homework W8/SQLQuery_HomeworkW8.sql

```diff
@@ -0,0 +1,340 @@
+CREATE DATABASE ZooManagemnt;
+
+USE ZooManagemnt;
+-- Creating tables
+CREATE TABLE ZoneArea
+(
+Id INT IDENTITY(1,1) PRIMARY KEY,
+Zone VARCHAR(50)
+);
+
+CREATE TABLE Species
+(
+Id INT IDENTITY(1,1) PRIMARY KEY,
+Name VARCHAR(50),
+Zone INT REFERENCES ZoneArea(Id)
+);
+
+CREATE TABLE Animals
+(
+Id INT IDENTITY(1,1) PRIMARY KEY,
+Species INT REFERENCES Species(Id)
+);
+
+CREATE TABLE Workers
+(
+Id INT IDENTITY(1,1) PRIMARY KEY,
+FirstName VARCHAR (20),
+LastName VARCHAR (20),
+WorkArea INT REFERENCES ZoneArea(Id)
+);
+
+CREATE TABLE Ticket
+(
+Id INT IDENTITY(1,1) PRIMARY KEY,
+Price MONEY,
+ValidityStart DATE,
```

Comment:
It is an interesting approach for validities but it needs some more validations if we want it to stay this way because the requirement said that there are three possible options (one day, one week, one month). With the current structure I can add a ticket that is valid for two weeks and five days. :)
A new table, maybe called, ValidityTypes could be used. Inside ValidityTypes we can store information like "one day", "one month", "one week" etc. and then it can be referenced in the Tickets table.